static/bidder-info/bidmachine.yaml

```diff
@@ -0,0 +1,7 @@
+maintainer:
+  email: hi@bidmachine.io
+capabilities:
```

Comment:
I see in the docs PR that Bidmachine is registered with IAB Europe for vendor 736. Please add that to this file using `gvlVendorID : 736`.